src/ControlSystem/Event.hpp

```diff
@@ -97,6 +101,13 @@ class Event : public ::Event {
           using Submeasurement = tmpl::type_from<decltype(submeasurement)>;
           db::apply<Submeasurement>(box, measurement_id, cache, array_index,
                                     component, ControlSystems{});
+          if (Parallel::get<Tags::Verbosity>(cache) >= ::Verbosity::Debug) {
+            Parallel::printf(
+                "%s, time = %s: Running control system events for measurement "
+                "'%s'.\n",
+                array_index, measurement_id,
```

Comment:
I think in tests we often use `ArrayIndex = int`, and that that will crash `printf` because it wants `%d` as a format specifier.  Maybe we don't care, but if we do wrapping in `get_output` should be safe.